### PR TITLE
Caroline & Pauline

### PR DESCRIPTION
**Pull request by Pauan8**
Movies

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -5,9 +5,226 @@ body {
...
+  flex-direction: column;
+}
+
+/* MovieCard */
+
+.movie-card {
+  position: relative;
+}
+
+.movie-card__image-container {
```
Comment (darcywinterhill) [inline code/src/index.css:30]: Clear and obvious classnames for your elements ✨

File: code/src/pages/PopularList.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+
+import MovieCard from "../components/MovieCard";
+
+const PopularList = () => {
+  const [popularList, setPopularList] = useState([]);
+  const API_URL =
```
Comment (darcywinterhill) [inline code/src/pages/PopularList.js:7]: A suggestion is to store the  API's in their own file to reduce the amount of code in the components/pages files 🏷

File: code/src/components/BackLink.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const BackLink = () => {
+  return (
+    <a className="back-link" href="/">
```
Comment (darcywinterhill) [inline code/src/components/BackLink.js:5]: Is there a reason why this is an < a  > and not a < Link >? 🐱‍👤

File: code/src/pages/MovieDetails.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+const MovieDetails = () => {
+  const { id } = useParams();
+  const [movie, setMovie] = useState([]);
+  const API_URL = `https://api.themoviedb.org/3/movie/${id}?api_key=9e2d991bdc86d11d2a7e1e110ef63aac&language=en-US`;
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((movie) => setMovie(movie))
+      .catch((error) => console.log(error));
```
Comment (darcywinterhill) [inline code/src/pages/MovieDetails.js:15]: What we know we shouldn't use console.log on deployed sites 🎆

**Review by darcywinterhill — APPROVED** [review]
Clean and easy-to-read responsive project. 
Super! 🎇✨

/Darcy & Ann-Sofi

File: code/src/App.js [hunk 5]
```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/src/App.js:2]: instead of typing BrowserRouter you just type Router in your code.

File: code/src/pages/PopularList.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    "https://api.themoviedb.org/3/movie/popular?api_key=9e2d991bdc86d11d2a7e1e110ef63aac&language=en-US&page=1";
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((movies) => setPopularList(movies.results))
+      .catch((error) => console.log(error));
```
Comment (KarolCislowski) [inline code/src/pages/PopularList.js:14]: 👮🏻

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Nice project but please remove all console outputs.

**Review by KarolCislowski — APPROVED** [review]

**Pull request closed without merging**